src/Pomodoro.Web/wwwroot/css/app.css

```
@@ -2369,6 +2337,44 @@ textarea.tep-input {
+    z-index: 1000;
+    box-shadow: 0 4px 20px rgba(0, 0, 0, 0.4);
+    animation: slideUp 0.3s ease;
+}
+
+@media (min-width: 900px) {
+    .app-toast {
+        bottom: 2rem;
+    }
+}
+
+@keyframes slideUp {
```

Stylelint (17.14.0)

[error] Expected keyframe name "slideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)